### ISSUE TITLE
[DISCUSSION] This project's future

### ISSUE THREAD

**Issue by NathanAP** [fawmi/vue-google-maps#159]
## Why this issue is here

In the issue #152 we were talking about this project not being maintained anymore. Right now we have 4 PR, in which fixes some small bugs and adds some features, like issue #125 (PR #145).

## My motivation

I'm following this project since the start of this year and this is the best Google Maps I've found so far. Since August, I notice that most of the open issues are not being answered anymore, so I've been following this repository since then.

## Why it should be forked by the community

- @fawmi made a great job and I'm grateful for him creating this repo and making it better since the start. This project belongs to him and there is no one who deserves more credits than him. The community should fork it and keep it active until Fawmi decides what to do with it;

- Accessing Fawmi's [profile](https://github.com/fawmi) you will find that he is inactive since July/2022. The coincidence is that it was when he updated this repo for the last time; 

- This repo is important and has a great potential;

## Who should fork it

IMO, anyone who have the motivation, the responsability and wants to make it better. If you want to do it, please remember that this project belongs to Fawmi and the community.

## Why I didn't forked it

TBH I would do it right now, but I believe all the community has a voice, so I want your opinion. Also, I never had a project forked or made something in this level of importance, so I would need help to keep the project maintained. 

I will do anything I can to help if someone decides to fork it.

edit: all my edits were made to fix the text.

**Comment by NathanAP** [comment]
I'm pinging some people to enter this conversation/decision. Feel free to not answer if you are not interested.

@websitevirtuoso, @tahaipek, @reslear, @Giuseppetm, @dmoyadev

**Comment by reslear** [comment]
of course, I suggest you calmly fork and work on

**Comment by websitevirtuoso** [comment]
Yes, I agree that somebody should work and the community can always help.
I can support it from time to time but don't want to be the owner because too busy due to full-time job and a small baby.
Go ahead and let's improve it

**Comment by NathanAP** [comment]
I've sent an email to Fawmi to explain the actual scenario. If he not answer this time I believe the best is to fork this project to keep it alive.

**Comment by tahaipek** [comment]
> Yes, I agree that somebody should work and the community can always help. I can support it from time to time but don't want to be the owner because too busy due to full-time job and a small baby. Go ahead and let's improve it

I have the same reasons. I will support this repository as much as I can.
Let's proceed according to @fawmi 's answer.

**Comment by KickSchutte67** [comment]
I just started working with this package and I would like more frequent updates. Go ahead :)

**Comment by NathanAP** [comment]
Hello again friends! Fawni didn't answered me. I decided today to fork this library to my own profile, you can access it [here](https://github.com/NathanAP/vue-google-maps-community-fork). 

I wrote in readme about everything I made and adapted to work just like it was intended. Please let me know if everything is working fine for you as well.

Also, you can find ways to contact me and the community in there as well. Good luck guys :) I'm sure we will keep this awesome project healthy <3
